packages/dd-trace/src/config.js

```diff
@@ -443,6 +443,12 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       5 // seconds
     )
 
+    const DD_APPSEC_SCA_ENABLED = coalesce(
+      options.sca && (options.sca === true || options.sca.enabled === true),
```

Comment:
could we omit the programmatic conf maybe ? is it in the spec ? having a "noop" env var is already humiliating enough as it is :laughing: